src/uct/ib/base/ib_device.c

```diff
@@ -27,6 +27,16 @@ static ucs_stats_class_t uct_ib_device_stats_class = {
 };
 #endif
 
+static uct_ib_device_info_t uct_ib_device_info_table[] = {
+  {0x02c9, 4099, "ConnectX-3",     UCT_IB_DEVICE_FLAG_MLX4_PRM, 10},
+  {0x02c9, 4103, "ConnectX-3 Pro", UCT_IB_DEVICE_FLAG_MLX4_PRM, 11},
+  {0x02c9, 4113, "Connect-IB",     UCT_IB_DEVICE_FLAG_MLX5_PRM, 20},
+  {0x02c9, 4115, "ConnectX-4",     UCT_IB_DEVICE_FLAG_MLX5_PRM, 30},
+  {0x02c9, 4117, "ConnectX-4 LX",  UCT_IB_DEVICE_FLAG_MLX5_PRM, 28},
+  {0x02c9, 4121, "ConnectX-5",     UCT_IB_DEVICE_FLAG_MLX5_PRM, 40},
+  {0,      0,    "Generic HCA",    0,                           0}
```

Comment:
Maybe add entry from env vars if exist.this will allow adding new card at runtime with vars w.o. Code changes.